galite-core/src/main/kotlin/org/kopi/galite/visual/ui/vaadin/form/DTextField.kt

```diff
@@ -274,11 +276,11 @@ open class DTextField(
     //---------------------------------------
     // IMPLEMENTATIONS
     //---------------------------------------
-    override fun toGui(modelTxt: String?): String? {
+    override fun toModel(modelTxt: String?): String? {
       return modelTxt
     }
 
-    override fun toModel(guiTxt: String?): String? {
+    override fun toGui(guiTxt: String?): String? {
```

Comment:
I don't see the point of these changes ?

Reply:
I made those changes just to  make the function names more meaningful and aligns them with their respective purposes

Reply:
I think there is a misunderstanding : 
ToGui function aims to transform a model text into a text to be interpreted by the GUI. In this case, the GUI text is the same as the model text this is why we return the model text value. 
The same logic applies to the function toModel.
You should restore these functions to their previous states.